CesiumUtility/test/TestJoinToString.cpp

```diff
@@ -0,0 +1,22 @@
+#include <CesiumUtility/joinToString.h>
+
+#include <catch2/catch.hpp>
+
+using namespace CesiumUtility;
+
+TEST_CASE("joinToString") {
```

Comment:
I think it's worth splitting this into multiple test cases with clearer labels to show what they're testing.